src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.margint_indent {
+  margin-top: 20px;
+}
+
+.search_form {
+  display: block;
+  position: relative;
+  height: 70px;
+  margin-bottom: 20px;
+}
+
+.search_svg {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.input_big_search_bar {
+  font-family: Avenir-book, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border-radius: 4px;
+}
+
+.input_big_search_bar::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.input_big_search_bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input_big_search_bar:focus {
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+  font-weight: 700;
+}
+
+.search_form_small {
+  display: block;
+  position: relative;
+  height: 42px;
+}
+
+.search_svg_small {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.input_small_search_bar {
```

Comment:
add some common style and apply repeated styles on it

Reply:
okey